src/ngraph/pass/opset1_downgrade.cpp

```diff
@@ -0,0 +1,125 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include <algorithm>
+
+#include "ngraph/node.hpp"
+#include "ngraph/ops.hpp"
+#include "ngraph/pass/opset1_downgrade.hpp"
+#include "ngraph/provenance.hpp"
+#include "ngraph/validation_util.hpp"
+
+using namespace std;
+using namespace ngraph;
+
+namespace
+{
+    shared_ptr<Node> op_cast(shared_ptr<op::v3::Broadcast> node)
+    {
+        const auto data = node->input_value(0).get_node_shared_ptr();
+        const auto target_shape = node->input_value(1).get_node_shared_ptr();
+
+        shared_ptr<Node> replacement_node;
+        switch (node->get_broadcast_spec().m_type)
+        {
+        case op::BroadcastType::BIDIRECTIONAL:
+        {
+            const auto const_filled_with_ones = make_shared<op::v1::Broadcast>(
+                op::Constant::create(data->get_element_type(), {}, {1}), target_shape);
+            replacement_node = make_shared<op::v1::Multiply>(data, const_filled_with_ones);
+            break;
+        }
+        case op::BroadcastType::EXPLICIT:
+        {
+            const auto axes_mapping = node->input_value(2).get_node_shared_ptr();
+            replacement_node = make_shared<op::v1::Broadcast>(
+                data, target_shape, axes_mapping, op::AutoBroadcastType::EXPLICIT);
+            break;
+        }
+        case op::BroadcastType::NUMPY:
+        {
+            replacement_node =
+                make_shared<op::v1::Broadcast>(data, target_shape, op::AutoBroadcastType::NUMPY);
+            break;
+        }
+        case op::BroadcastType::PDPD:
+        {
+            op::AutoBroadcastSpec broadcast_spec;
+            broadcast_spec.m_type = op::AutoBroadcastType::PDPD;
+            broadcast_spec.m_axis = node->get_broadcast_spec().m_axis;
+            replacement_node = make_shared<op::v1::Broadcast>(data, target_shape, broadcast_spec);
+            break;
+        }
+        }
```

Comment:
IMHO It is a good practice to add `default` statement to catch unhandled enum values. Moreover lack of it would come up as an issue in any tool used to check the code during SDL process.